### DISCUSSION TITLE
Practica CSS Terminada

### DISCUSSION THREAD

**Discussion by Rodrigorenteria** [Launch-X-Latam/MisionFrontEnd#373] [Show and tell]
Lo intente hacer lo más parecido posible, espero les guste.

[Vaccination](https://rodrigorenteria.github.io/Vaccination/)

**Comment by OscarJG20** [comment]
Te quedo bastante bien en versión de escritorio.
Creo que como reto estaría bueno que ahora sea responsive.
Ánimo!

**Reply by Rodrigorenteria** [reply]
Si, justo eso me faltó pero esta semana la termino de hacer responsive solo que ya la quería subir para no atrásame tanto jaja
Muchas gracias!
